routes/api.js

```diff
@@ -98,14 +120,68 @@ router.get('/getProgressStatus', (req, res) => {
         });
 });
 
+/**
+ * GET mail policy API
+ * @param  {String} '/mailpcpolicy'                                   URI of the resource
+ * @param  {Function} authenticationHelpers.isAuthOrRedirect          Check if user is authenticated else redirect him back to login
+ * @param  {Function} (req, res)                                      Anonymous function to handle request and response
+ */
 router.get('/mailpcpolicy', authenticationHelpers.isAuthOrRedirect, (req, res) => {
     const email = req.user.email;
     mail.mailOptions.to = email;
     mail.mailOptions.subject = 'Peace Corps Policy';
     mail.mailOptions.html = '<H2> Peace Corps Policy </H2>';
     mail.smtpTransport.sendMail(mail.mailOptions, function(error) {
         error ? res.status(500).json({error: 'Something Went Wrong! Try again later.'}) : res.json({message: 'Mail Sent Succesfully.'});
-    })
+    });
+});
+
+/**
+ * UPDATE progress status API
+ * @param  {String} '/updateProgressStatus'                           URI of the resource
+ * @param  {Function} authenticationHelpers.isAuthOrRedirect          Check if user is authenticated else redirect him back to login
+ * @param  {Function} (req, res)                                      Anonymous function to handle request and response
+ */
+router.patch('/updateProgressStatus', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    if(req.body && req.body.stage && req.body.activity) {
+        const currStage = req.body.stage;
+        const currActivity = req.body.activity;
+        localUser.find({
+            where: {
+                email: req.user.email
+            },
+            include: [progress]
+        }, {raw: true})
+            .then(data => {
+                const progressStage = data.progress.stage;
+                const progressActivity = data.progress.activity;
+                const stageDiff = currStage-progressStage;
+                const activityDiff = currActivity-progressActivity;
+                if((currStage===progressStage || stageDiff===1) && (activityDiff===1)){
```

Comment:
let's have space around these operators (and probably a good idea to add a rule to eslint), mostly to stay consistent with https://github.com/systers/PC-Prep-Kit/pull/28/files/2ba402e83f8b231321bf8d89db0124d61dbddf6c#r129378122